guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -83,8 +83,11 @@
 
 .login-ui .login-dialog .logo {
     display: block;
-    max-width: 3em;
     margin: 0.5em auto;
+    width: 3em;
+    height: 3em;
+    background-size: 3em;
```

Comment:
Need a `-moz` and `-webkit` for these, I think ... see: https://github.com/glyptodon/guacamole-client/blob/fd3619a41d0d7dbcd8ef4c1086b817bbb927a204/guacamole/src/main/webapp/app/client/styles/transfer.css#L77-L80